lib/DraggableCore.es6

```diff
@@ -331,6 +345,10 @@ export default class DraggableCore extends React.Component {
     // We're on a touch device now, so change the event handlers
     dragEventFor = eventsFor.touch;
 
+    // Remove the event listener that stopped document scrolling
+    document.removeEventListener('touchmove', this.removeScroll, false);
```

Comment:
In fact, why don't we just add/remove this listener `onTouchStart` outright? So no `state.scrolling` is required at all.

Also be sure to add an extra `document.removeEventListener` on unmount in case an unmount happens while dragging.